EcommerceStore.Application/Exceptions/ExceptionMessages.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace EcommerceStore.Application.Exceptions
+{
+    public static class ExceptionMessages
+    {
+        public static string BrandNotFound { get; private set; } = "Brand with the specified data was not found";
```

Comment:
you can make exception messages a bit more meaningful and add additional details (brandId, brandName)